pkg/runner/test_runner.go

```diff
@@ -738,36 +754,9 @@ func (tr *TestRunner) Run(cancel, pause <-chan struct{}, t *test.Test, targets [
 		log.Printf("TestRunner completed")
 	}
 
-	ev := storage.NewTestEventEmitterFetcher(testevent.Header{
```

Comment:
this has been moved to WaitPipelineCompletion, the function that has visibility of all the necessary information to emit this event